media-libs/cimg/cimg-3.1.6.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit git-r3
+
+DESCRIPTION="C++ template image processing toolkit"
+HOMEPAGE="https://github.com/GreycLab/CImg"
+EGIT_REPO_URI="https://github.com/GreycLab/CImg.git"
+EGIT_COMMIT="v.${PV}"
```

Comment:
Please download and use a tarball instead. Allows you to not use *git-r3* eclass too.